script/release/bin/README.md

```diff
@@ -30,14 +30,11 @@ This script just outputs the theoretical "next" version that a release would use
 This script creates all the requisite tags and CI builds that will populate required release assets.
 
 * Creates the git tag
-* Kicks off all release builds on AppVeyor and GitHub Actions
+* Kicks off all release builds on GitHub Actions
 
 ### `run-release-build`
 
-This script is used to re-kick specific release builds after they fail. Sudowoodo is responsible for prompting the release team as to whether or not to run this script. It's currently only used for AppVeyor builds.
-
-> [!IMPORTANT]
-> This script should be removed and the "rerun" logic for AppVeyor be implemented in Sudowoodo specifically in the same way that GitHub Actions' rerun logic is.
+This script is used to re-kick specific release builds after they fail. Sudowoodo is responsible for prompting the release team as to whether or not to run this script.
```

Comment:
I'm not sure about the accuracy of this section going forward - it says it's only used for AppVeyor builds but it seems `run-release-builds` is still being used and was updated in #45137?

Reply:
I don't even think we really use this script manually to rekick build anymore, tbh 🤔 I do think it works with GHA as well, though, let me test it and we can either remove it or update this language